var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -193,7 +193,7 @@ def setup_build_environment(self, env):
 
         # https://github.com/scipy/scipy/issues/14935
         if self.spec.satisfies("%intel ^py-pythran") or self.spec.satisfies("%oneapi ^py-pythran"):
-            if self.spec["py-pythran"].version < Version("0.12"):
+            if self.spec["py-pythran"].version < Version("0.13"):
```

Comment:
If we don't know that this will be fixed in 0.14 then we might as well remove the version check. Personally I would open an issue with the scipy/pythran devs so we at least have something to reference so we can track when it gets fixed.

Reply:
How about this compromise: I change the first conditional to `%intel` *only* (i.e. eliminate the `%oneapi` test) and then remove the `Version()` check altogether, as you propose.  This problem gets resolved for Intel Classic in perpetuity as it completes its supported lifecycle by the end of this calendar year; we re-visit any `%oneapi` issues more vigor if/when it arises in the future.

Reply:
I'm fine with that. I would still strongly encourage you to open an issue though, otherwise this will never be fixed.

Reply:
Pushing this proposal.  Our code team is moving to `%oneapi` and we will definitely engage upstream to open an issue if/when we encounter it with the new toolchain.